ci/pipeline.tf

```diff
@@ -0,0 +1,26 @@
+module "pipeline" {
+  source = "../modules//platsec_ci_terraform_pipeline"
+
+  pipeline_name = "${local.repo}-pipeline"
+  src_repo      = local.repo
+  branch        = "PSEC-2203-ci" # "main"
```

Comment:
Does this need switching to main yet?

Reply:
yes... will follow up with a new PR right after this one